Added Tetris game

rumbleFTW opened the pull request:
# Added Tetris game in the ./Tetris directory

SiddharthaBhattacharjee commented:
Please deploy your branch on GitHub server and comment the link here , we need to see the hosted page to review your Pull Request.

rumbleFTW commented:
- [Website preview](https://rumbleftw.github.io/TimePassGames/Tetris/index.html)
- Added Details to CONTRIBUTORS.md
- Removed duplicate GuessTheNumber entries

SiddharthaBhattacharjee commented:
Thank You for contributing, your game is nice, however, there are issues like there being no touchscreen controls, we are merging your PR and we will make some new issues for fixing the issues with the game. Also, there might be some design changes made to the game in the future.
Please try to make another PR addressing the issue which will be created about the touch screen support of this game.
